src/Swarrot/Processor/Sentry/SentryProcessor.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Swarrot\Processor\Sentry;
+
+use Swarrot\Broker\Message;
+use Swarrot\Processor\ProcessorInterface;
+
+class SentryProcessor implements ProcessorInterface
+{
+    /**
+     * @var ProcessorInterface
+     */
+    private $processor;
+
+    /**
+     * @var \Raven_Client
+     */
+    private $client;
+
+    /**
+     * @param ProcessorInterface $processor
+     * @param \Raven_Client      $client
+     */
+    public function __construct(ProcessorInterface $processor, \Raven_Client $client)
+    {
+        $this->processor = $processor;
+        $this->client = $client;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function process(Message $message, array $options)
+    {
+        try {
+            return $this->processor->process($message, $options);
+        } catch (\Throwable $e) {
+            $this->handleException($e, $message, $options);
+        } catch (\Exception $e) {
+            $this->handleException($e, $message, $options);
+        }
+    }
+
+    /**
+     * @param \Throwable|\Exception  $exception
+     * @param Message                $message
+     * @param array                  $options
+     *
+     * @throws \Throwable|\Exception
+     */
+    private function handleException($exception, Message $message, array $options)
+    {
+        $properties = $message->getProperties();
+
+        $data = [
```

Comment:
This is quite static if users need more tags like a specific header, they would have to create a fully new Processor.
I would at least move the `$data` array construction in a protected method, to stay extendable. Being able to configure which properties to use as tag would be great :)